content/docs/operating/configuration.md

```diff
@@ -662,6 +662,12 @@ See below for the configuration options for Marathon discovery:
 servers:
   - <string>
 
+# Optional bearer token authentication information.
+  [ bearer_token: <string> ]
```

Comment:
This code snippet describes a single <marathon_sd_config>, not a list like your example. Your current indentation puts it under the `servers` key, which is both wrong and impossible.